webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-readonly.js

```diff
@@ -44,6 +48,15 @@ export class SakaiRubricReadonly extends RubricsElement {
             <span class="hidden-sm hidden-xs sr-only"><sr-lang key="copy" /></span>
             <span role="button" title="${tr("copy_to_site", [this.rubric.title])}" tabindex="0" class="clone fa fa-copy" @click="${this.copyToSite}"></span>
           </div>
+          ${this.exportPdf=='true' ? html`
+          <div class="action-container">
+            <span class="hidden-sm hidden-xs sr-only"><sr-lang key="export_label" /></span>
```

Comment:
Weird logic and using strings instead of boolean.

As Adrian mentioned do not use the "? X : Y" logic, just use Litelement templates.